index.js

```diff
@@ -1,57 +1,79 @@
 'use strict'
 
-var level = require('level')
-var has = require('has')
```

Comment:
Are we still using `has` elsewhere, or can it be removed from `package.json`?

Reply:
I can make sure and then remove